.github/renovate.json

```
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

[Formatting] Heh, sorry, my suggestion wasn't quite what the rest is formatted as...

```suggestion
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```